tests/ci-commitmessage-submodules.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euo pipefail
+
+BUMP_SUBMODULE_REGEXP="Update submodule: "
```

Comment:
I agree that all sounds better...this is another case where I'm not totally sure it's was quite worth the time to polish, but I did anyways.  At least it's more likely to be reusable by others.  Update ⬇️